sdk/python/foundation-models/system/inference/text-generation/falcon-safe-online-deployment.ipynb

```diff
@@ -107,31 +105,40 @@
     "except Exception as ex:\n",
     "    # Fall back to InteractiveBrowserCredential in case DefaultAzureCredential not work\n",
     "    credential = InteractiveBrowserCredential()"
-   ]
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1706764189401
+    }
+   }
   },
   {
    "attachments": {},
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "#### 1.5 Configure workspace "
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "from azure.ai.ml import MLClient\n",
     "\n",
     "try:\n",
-    "    ml_client = MLClient.from_config(credential=credential)\n",
+    "    subscription_id = \"79a1ba0c-35bb-436b-bff2-3074d5ff1f89\"\n",
+    "    resource_group = \"qianie-rg\"\n",
+    "    workspace_name = \"qianie-westus3\"\n",
+    "\n",
+    "    # get a handle to the workspace\n",
+    "    ml_client = MLClient(credential, subscription_id, resource_group, workspace_name)\n",
     "except Exception as ex:\n",
     "    # enter details of your AML workspace\n",
-    "    subscription_id = \"<SUBSCRIPTION_ID>\"\n",
-    "    resource_group = \"<RESOURCE_GROUP>\"\n",
-    "    workspace = \"<AML_WORKSPACE_NAME>\"\n",
+    "    subscription_id = \"79a1ba0c-35bb-436b-bff2-3074d5ff1f89\"\n",
```

Comment:
you probably don't want everyone use your subscription :)

Reply:
Good catch!